Zotero/Models/Database/Database.swift

```diff
@@ -44,6 +44,32 @@ struct Database {
             if schemaVersion < 36 {
                 self.migrateTagNames(migration: migration)
             }
+            if schemaVersion < 37 {
+                self.extractItemHtmlFreeContent(migration: migration)
+            }
+        }
+    }
+
+    private static func extractItemHtmlFreeContent(migration: Migration) {
+        migration.enumerateObjects(ofType: RItem.className()) { oldObject, newObject in
+            guard let rawType = oldObject?["rawType"] as? String, let fields = oldObject?["fields"] as? List<MigrationObject> else { return }
+
+            let content: String
+
+            switch rawType {
+            case ItemTypes.note:
+                guard let _content = fields.first(where: { $0["key"] as? String == FieldKeys.Item.note })?["value"] as? String, !_content.isEmpty else { return }
+                content = _content.strippedHtmlTags
+
+            case ItemTypes.annotation:
+                guard let _content = fields.first(where: { $0["key"] as? String == FieldKeys.Item.Annotation.comment })?["value"] as? String, !_content.isEmpty else { return }
+                content = _content.strippedHtmlTags
```

Comment:
Is the annotation comment in the previous scheme html?

Reply:
No, I forgot to change it there, good catch 👍

Reply:
https://github.com/zotero/zotero-ios/pull/723/commits/376e67346d30e37e2f30474a47baec17ee3b4e32